warnings when running gemini cli with deno

but it seems to work with no hard errror

Version: deno 2.7.6+42eb6ab (canary, release, x86_64-unknown-linux-gnu)

```
mrcool@1E-55-11-07-05-2E ~/d/d/cf (master)> deno -A npm:@google/gemini-cli
Keychain initialization encountered an error: Cannot find module '../build/Release/keytar.node'
Require stack:
- /home/mrcool/dev/deno/cf/node_modules/.deno/keytar@7.9.0/node_modules/keytar/lib/keytar.js
- /home/mrcool/dev/deno/cf/node_modules/.deno/keytar@7.9.0/node_modules/keytar/lib/keytar.js
Using FileKeychain fallback for secure storage.
Loaded cached credentials.
Keychain initialization encountered an error: Cannot find module '../build/Release/keytar.node'
Require stack:
- /home/mrcool/dev/deno/cf/node_modules/.deno/keytar@7.9.0/node_modules/keytar/lib/keytar.js
- /home/mrcool/dev/deno/cf/node_modules/.deno/keytar@7.9.0/node_modules/keytar/lib/keytar.js
Using FileKeychain fallback for secure storage.
Loaded cached credentials.
Failed to write terminal capability queries: TypeError [ERR_INVALID_ARG_TYPE]: The "fd" argument must be of type number. Received undefined
    at __node_internal_ (ext:deno_node/internal/validators.mjs:134:15)
    at __node_internal_ (ext:deno_node/internal/fs/utils.mjs:909:3)
    at Module.writeSync (node:fs:1184:8)
    at file:///home/mrcool/dev/deno/cf/node_modules/.deno/@google+gemini-cli@0.34.0/node_modules/@google/gemini-cli/dist/src/ui/utils/terminalCapabilityManager.js:220:12
    at new Promise (<anonymous>)
    at TerminalCapabilityManager.detectCapabilities (file:///home/mrcool/dev/deno/cf/node_modules/.deno/@google+gemini-cli@0.34.0/node_modules/@google/gemini-cli/dist/src/ui/utils/terminalCapabilityManager.js:117:12)
    at setupTerminalAndTheme (file:///home/mrcool/dev/deno/cf/node_modules/.deno/@google+gemini-cli@0.34.0/node_modules/@google/gemini-cli/dist/src/utils/terminalTheme.js:30:37)
    at main (file:///home/mrcool/dev/deno/cf/node_modules/.deno/@google+gemini-cli@0.34.0/node_modules/@google/gemini-cli/dist/src/gemini.js:662:11) {
  code: 'ERR_INVALID_ARG_TYPE',
  toString: [Function (anonymous)]
}
```

--- a/ext/node/polyfills/process.ts
+++ b/ext/node/polyfills/process.ts
@@ -1317,6 +1317,9 @@ internals.__bootstrapNodeProcess = function (
     if (io.stdout.isTerminal()) {
       /** https://nodejs.org/api/process.html#process_process_stdout */
       stdout = process.stdout = new TTYWriteStream(1);
+      // For supporting legacy API we put the FD here.
+      // Ref: https://github.com/nodejs/node/blob/main/lib/internal/bootstrap/switches/is_main_thread.js
+      stdout.fd = 1;
       // Match Node.js: stdio streams are indestructible.
       // Libraries like mute-stream (@inquirer/prompts) call destroy()/end()
       // on process.stdout between prompts. Without this, the underlying TTY
@@ -1343,6 +1346,8 @@ internals.__bootstrapNodeProcess = function (
     if (io.stderr.isTerminal()) {
       /** https://nodejs.org/api/process.html#process_process_stderr */
       stderr = process.stderr = new TTYWriteStream(2);
+      // For supporting legacy API we put the FD here.
+      stderr.fd = 2;
       stderr._isStdio = true;
       stderr.destroySoon = stderr.destroy;
       stderr._destroy = function (err, cb) {

--- a/tests/specs/node/tty_stream_fd/__test__.jsonc
+++ b/tests/specs/node/tty_stream_fd/__test__.jsonc
@@ -1,0 +1,8 @@
+{
+  "tests": {
+    "tty_stream_fd": {
+      "args": "run main.mjs",
+      "output": "main.out"
+    }
+  }
+}

--- a/tests/specs/node/tty_stream_fd/main.mjs
+++ b/tests/specs/node/tty_stream_fd/main.mjs
@@ -1,0 +1,12 @@
+import process from "node:process";
+import fs from "node:fs";
+
+// process.stdout.fd and process.stderr.fd should always be set,
+// regardless of whether the stream is a TTY or not.
+// This is needed by libraries like @google/gemini-cli that use
+// fs.writeSync(process.stdout.fd, data) for terminal capability detection.
+console.log("stdout.fd:", typeof process.stdout.fd, process.stdout.fd);
+console.log("stderr.fd:", typeof process.stderr.fd, process.stderr.fd);
+
+// fs.writeSync with process.stdout.fd should work
+fs.writeSync(process.stdout.fd, "writeSync works\n");

--- a/tests/specs/node/tty_stream_fd/main.out
+++ b/tests/specs/node/tty_stream_fd/main.out
@@ -1,0 +1,3 @@
+stdout.fd: number 1
+stderr.fd: number 2
+writeSync works

